cmd/util/cmd/read-hotstuff/cmd/root.go

```diff
@@ -30,6 +32,11 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&flagDatadir, "datadir", "d", "/var/flow/data/protocol", "directory to the badger dababase")
 	_ = rootCmd.MarkPersistentFlagRequired("datadir")
 
+	rootCmd.PersistentFlags().StringVar(&flagPebbleDir, "pebble-dir", "", "directory to the pebble dababase")
+
+	rootCmd.PersistentFlags().StringVar(&flagChain, "chain", "", "Chain name, e.g. flow-mainnet, flow-testnet")
```

Comment:
previously, we used to get the ChainID from the root block. Now we need to provide it as a command line argument. No significant concerns, just curious what the reasoning is ...

Reply:
[I mentioned here](https://github.com/onflow/flow-go/pull/7358#discussion_r2067195214). Basically, it allows us to remove the dependency on protocol state, which was only used to query the chain id. 

Since the protocol state is still coupled with badger database, then the read-hotstuff util can not be using the new storage API.

Reply:
sorry, I overlooked your comment in the beginning. Thanks for the explanation 👍